moh-hni-bathl7/pom.xml

```diff
@@ -64,22 +63,21 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<version>2.3.2</version>
+				<version>3.8.1</version>
```

Comment:
updated plugin versions to be more recent (and in line with what hns-esb and hns-client use)